app/services/facebook_service.rb

```diff
@@ -0,0 +1,9 @@
+class FacebookService
+  def initialize(access_token)
+    @access_token = access_token
+  end
+
+  def profile
+    Koala::Facebook::API.new(@access_token).get_object('me?fields=gender,name')
```

Comment:
I would create a `client` in the `initialize` method and then call it here. Something like:
```
class FacebookService
  attr_reader :client

  def initialize(access_token)
    @client = Koala::Facebook::API.new(access_token)
  end

  def profile
    client.get_object('me?fields=gender,name')
  end
end
```
Also, use `attr_reader` for the `client` so you can avoid the `@`.

WDYT @GuilleLeopold, @mdornel and @JuanfraM ?